src/jsx.d.ts

```diff
@@ -273,24 +273,30 @@ export namespace JSXInternal {
 	interface PathAttributes {
 		d: string;
 	}
+	
+	type EventHandlers<E extends Event> = EventHandler<E> | UnistoreEventHandler<any, E>;
```

Comment:
Thanks for your PR 🎉 I'm honestly uneasy with adding library specific code to `preact`. If we start with it soon others will follow and we'll be forced to maintain it. Maybe there is a way to fix this in the typings for `unistore`? That would be a lot cleaner 🙂